src/usdb_syncer/gui/comment_dialog.py

```diff
@@ -0,0 +1,34 @@
+"""Dialog to post a comment on one or multiple songs."""
+
+from typing import Callable, Iterator
+
+from PySide6.QtWidgets import QDialog, QWidget
+
+from usdb_syncer.gui.forms.CommentDialog import Ui_Dialog
+from usdb_syncer.usdb_scraper import RequestMethod, get_usdb_page
+from usdb_syncer.usdb_song import UsdbSong
+
+
+class CommentDialog(Ui_Dialog, QDialog):
+    """Dialog to post a comment on one or multiple songs."""
+
+    def __init__(
+        self, parent: QWidget, selected_songs: Callable[[], Iterator[UsdbSong]]
+    ) -> None:
+        super().__init__(parent=parent)
+        self._parent = parent
+        self._selected_songs = selected_songs
+        self.setupUi(self)
+
+    def accept(self) -> None:
+        for song in self._selected_songs():
+            payload = {"text": self.line_edit_username.text(), "stars": "neutral"}
+
+            get_usdb_page(
+                "index.php",
+                RequestMethod.POST,
+                headers={"Content-Type": "application/x-www-form-urlencoded"},
+                params={"link": "detail", "id": str(song.song_id), "comment": str(1)},
```

Comment:
This might need to be `str(int(song.song_id))` if leading 0s cause problems.

Reply:
I'll test this out, thanks for the good catch

Reply:
This sends the request with leading zero's. It's not a problem. So now it becomes a matter of what we want? I do see that USDB itself doesn't use the leading zero's in their routing.

Reply:
Changed it in the end, since usdb can't load the cover when sending leading zeros